google/cloud/documentai_toolbox/converters/vision_helpers.py

```diff
@@ -24,6 +24,8 @@
 from google.cloud.vision import TextAnnotation, Symbol, Word, Paragraph, Block, Page
 from google.cloud import vision
 
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
```

Comment:
I noticed that we are not very consistent with importing modules versus importing names/classes.  The general preference is to import modules.  For example the next engineer has to refer to the import line to know if `Page` here is from the Toolbox itself or from `google.cloud.vision`.

(This does not have to be done in this PR, but please add a internal clean up tracking issue to fix this.)